discretisedfield/field.py

```diff
@@ -2800,170 +2803,30 @@ def write(self, filename, representation="bin8", extend_scalar=False):
         True
         >>> os.remove(filename)  # delete the file
 
-        .. seealso:: :py:func:`~discretisedfield.Field.fromfile`
+        See also
+        --------
+        ~discretisedfield.Field.fromfile
+        ~discretisedfield.io.field_to_ovf
+        ~discretisedfield.io.field_to_vtk
+        ~discretisedfield.io.field_to_hdf5
 
         """
-        if filename.endswith((".omf", ".ovf", ".ohf")):
-            self._writeovf(
-                filename, representation=representation, extend_scalar=extend_scalar
+        filename = pathlib.Path(filename)
+        if filename.suffix in [".omf", ".ovf", ".ohf"]:
+            io.field_to_ovf(
```

Comment:
Why have you not passed the value of `save_subregions` argument to `write` method to the `io` module functions? For example,
```Python
io.field_to_hdf5(self, filename, save_subregions)
```
Am i missing something?

Reply:
You are right, this is missing.